File: app/src/main/java/jgappsandgames/smartreminderslite/sort/date/DayAdapter.java
package jgappsandgames.smartreminderslite.sort.date;

// Java
import java.util.Calendar;

// App
import jgappsandgames.smartreminderslite.adapter.TaskAdapterInterface;

// Save
import jgappsandgames.smartreminderssave.date.DateManager;

/**
 * DayAdapter
 * Created by joshua on 10/9/17.
 */
class DayAdapter extends TaskAdapterInterface {
    // Initializer
    DayAdapter(DayActivity activity, Calendar date_active) {
        super(activity, activity, DateManager.getDay(date_active));
    }
}

File: app/src/main/java/jgappsandgames/smartreminderslite/sort/date/DayAdapter.kt
package jgappsandgames.smartreminderslite.sort.date

// Java
import java.util.Calendar

// App
import jgappsandgames.smartreminderslite.adapter.TaskAdapterInterface

// Save
import jgappsandgames.smartreminderssave.date.DateManager

/**
 * DayAdapter
 * Created by joshua on 12/27/2017.
 */
class DayAdapter(activity: DayActivity, date_active: Calendar):
        TaskAdapterInterface(activity, activity, DateManager.getDayTasks(date_active))